CMakeLists.txt

```diff
@@ -217,6 +217,8 @@ if (SIGNED_JAR)
     add_custom_command(
         OUTPUT ${ACCP_JAR}
         COMMAND ${CMAKE_COMMAND} -E copy ${SIGNED_JAR} ${ACCP_JAR}
+        COMMAND ${CMAKE_COMMAND} -E echo Copied ${SIGNED_JAR} to ${ACCP_JAR}
+        COMMAND ${CMAKE_COMMAND} -E copy ${OPENSSL_CRYPTO_LIBRARY} $<TARGET_FILE_DIR:amazonCorrettoCryptoProvider>
```

Comment:
why do we need to copy aws-lc over? don't we bundle it in the ACCP JAR?

Reply:
This is for the test when we load the .so files from an external source (not the JAR), so libcrypto must be in the same directory as libaccp.